pkg/skaffold/deploy/cloudrun/status.go

```diff
@@ -344,9 +344,13 @@ func (r *runServiceResource) getTerminalStatus(crClient *run.APIService) (*run.G
 	return ready, nil
 }
 func (r *runServiceResource) reportSuccess() {
-	if r.url != "" {
-		eventV2.CloudRunServiceReady(r.path, r.url, r.latestRevision)
+	url := r.url
+	if url == "" {
+		// a URL may not be present if the default URL is disabled. Use - instead of empty in case anyone is parsing the
+		// event status
+		url = "-"
```

Comment:
It's not obvious that the URL can be not URL, but "dash". what if someone uses it like this:
```
if r.url != "" {
   ....
}
```
the condition will be true, because the URL is not empty, but contains "-"

Reply:
ok, keeping it as empty